uiowa/uiowa
# Hotfix: Updated to new config entity filter function

Resolves an issue with people list not loading because of a function being called that doesn't exist.

# How to test

```
ddev blt ds && ddev drush @default.local uli node/1/layout
```
- Confirm that you can add a block and that the person types filter works.
